flax/training/checkpoints.py

```diff
@@ -208,3 +211,51 @@ def read_chunk(i):
       return serialization.msgpack_restore(checkpoint_contents)
     else:
       return serialization.from_bytes(target, checkpoint_contents)
+
+
+def convert_pre_linen(params):
```

Comment:
Did you intent to only convert pre-linen param trees? Because I think this code will not handle the old Collections

Reply:
Added comment to the pydoc.